http/src/routing.rs

```diff
@@ -2157,6 +2171,33 @@ impl Display for Route<'_> {
 
                 f.write_str("/bans")
             }
+            Route::GetBansWithParameters {
```

Comment:
regarding the behavior of this function, there is another part of our codebase that enforces `before` taking precedence over `after`. should this code be updated to behave like that code, or should that code be simplified? https://github.com/twilight-rs/twilight/blob/e642b77704a7ad6453f5c5e9a903d0f55da08e26/http/src/routing.rs#L2302-L2338

Reply:
We should probably simplify the other code and just throw everything at Discord, letting it decide how it wants to take precedence.